web/src/components/netflow-topology/styles/styleNode.tsx

```diff
@@ -40,23 +42,25 @@ export enum DataTypes {
 }
 const ICON_PADDING = 20;
 
+type NodePeer = Node<NodeModel, Decorated<NodeData>>;
+
 type StyleNodeProps = {
-  element: Node;
+  element: NodePeer;
   getCustomShape?: (node: Node) => React.FC<ShapeProps>;
-  getShapeDecoratorCenter?: (quadrant: TopologyQuadrant, node: Node, radius?: number) => { x: number; y: number };
+  getShapeDecoratorCenter?: (quadrant: TopologyQuadrant, node: NodePeer, radius?: number) => { x: number; y: number };
   showLabel?: boolean;
   showStatusDecorator?: boolean;
-  regrouping?: boolean;
+  // TODO / TO CHECK: is this used? (not declared in BaseNodeProps) regrouping?: boolean;
```

Comment:
@jpinsonneau what to do here? is "regrouping" something used?

Reply:
Yes this is used. It's part of [nodeDragSourceSpec](https://github.com/patternfly/patternfly-react/blob/main/packages/react-topology/src/components/factories/components/componentUtils.ts#L97) since we use `withDragNode` in `stylesComponentFactory`.

Reply:
Restored: 4b1f878c6358fff0dd7f07d80a4c4155fab2e1ad